backend/src/Service/Order/OrderService.php

```diff
@@ -384,11 +384,17 @@ public function acceptedOrderByCaptainId($captainId): ?array
         return $response;
     }
 
-    public function getSpecificOrderForCaptain(int $id, int $userId): ?SpecificOrderForCaptainResponse
```

Comment:
Why you deleted "?" ?
When there is no such order with the passed id then **null** will be returned by the function.